src/LockFileException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Mesavolt\FiveOhThree;
+
+
+/**
+ * Exception thrown by the LockGuard when a lock file is detected
+ */
+class LockFileException extends \Exception
```

Comment:
I'd extend a more specific SPL exception type such as `RuntimeException` instead of using `Exception`, what do you think?

Reply:
done in eb24484c0fc73b34ccd2956f61c0df230f0552a6